frontend/components/styles/Buttons.tsx

```diff
@@ -44,20 +45,29 @@ export const Toggle = s.div`
   height: 2.5rem;
 `
 
-export const ToggleOption = s.button<{ active: boolean }>`
-  border-width: 0;
-  background-color: ${(props) =>
-    props.active ? colors.MEDIUM_BLUE : colors.LIGHTER_GRAY};
-  color: ${(props) => props.active && 'white'};
-  border-radius: 100px;
-  padding: 0.25rem 1rem;
-  outline: none;
+export const ToggleOption = React.forwardRef<
+  HTMLButtonElement,
+  { active: boolean } & React.ButtonHTMLAttributes<HTMLButtonElement>
+>(({ active, className, ...props }, ref) => (
+  <button
+    ref={ref}
+    type="button"
+    className={`
+      border-0
+      ${active ? 'bg-blue-500 text-white' : 'bg-neutral-100 text-gray-700'}
```

Comment:
again lets be consistent with our colors. I think we should just take the colors. Either we use colors.whatever or only tailwind. Probably prefer adding the colors we need to the config?

Reply:
Sounds good.